CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## To Be Released
 
+* feat(Probe interface): add support for context in the Check method
```

Comment:
No capitalization letter in the context, no? and better with hyphen instead of space (sorry i'm picky 😬 )

I mean: `feat(probe-interface): ...`

Reply:
And this one @stopostit?

Reply:
Missed it, sorry

Reply:
fixed